kube/src/api/mod.rs

```diff
@@ -35,3 +35,6 @@ pub use self::object::{Object, ObjectList, WatchEvent};
 
 mod metadata;
 pub use self::metadata::{ListMeta, ObjectMeta, Resource, TypeMeta};
+
+mod admission;
+pub use admission::{AdmissionRequest, AdmissionResponse, AdmissionReview, Operation, PatchType};
```

Comment:
We aren't actually adding a dependency on `json_patch` in `kube-rs` just yet since the response struct wants a serialized `Vec<u8>`. Though, adding a builder for this and taking the typed `JsonPatch` struct would be much better. I was hoping to push this to a follow up pr.